### PR TITLE
Make alert view scrollable

### PR DESCRIPTION
**Pull request by rablador**
Alert views should be scrollable, which is especially important for the change log.

- - -
This change is [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5354)

### CODE REVIEW

**Review by pinkisemils — COMMENTED** [review]
Would you mind posting a screenshot/screenrecording for this UI change? :slightly_smiling_face: 

> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5354#-:-NhW7Q4ABJtCX5v0X-fG:bjphte6)* status: 0 of 2 files reviewed, all discussions resolved

**Review by rablador — COMMENTED** [review]
Sure thing, will post one soon. 👍

> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5354#-:-NhWJ4FUAx_HlfJyw4gy:b-llc2pn)* status: 0 of 2 files reviewed, all discussions resolved

File: ios/MullvadVPN/View controllers/Alert/AlertViewController.swift [hunk 1]
```diff
@@ -86,61 +107,97 @@ class AlertViewController: UIViewController {
...
+        }
+    }
+
+    private func adjustButtonMargins() {
+        if !buttonView.arrangedSubviews.isEmpty {
+            // The presence of a button should yield a custom top margin.
+            buttonView.directionalLayoutMargins.top = UIMetrics.CustomAlert.interContainerSpacing
+
+            // Buttons below scrollable content should have more margin.
+            if scrollView.contentSize.height > scrollView.bounds.size.height  {
```
Comment (MrChocolatine) [inline ios/MullvadVPN/View controllers/Alert/AlertViewController.swift:195]:
```suggestion
            if scrollView.contentSize.height > scrollView.bounds.size.height {
```

**Review by mojganii — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5354#-:-Nh_nxrw2fV_RTgRPdrj:bnfp4nl)* status: 1 of 2 files reviewed, 1 unresolved discussion (waiting on @rablador)

**Review by rablador — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5354)* status: 1 of 2 files reviewed, 2 unresolved discussions (waiting on @MrChocolatine)

*[a discussion](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5354#-NhaW_MU4DX806MUXrz_:-NhaW_MU4DX806MUXrza:b-1bchm8) (no related file):*
Refactored away the setup method and improved encapsulation.

___
___
*[`ios/MullvadVPN/View controllers/Alert/AlertViewController.swift` line 195 at r2](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5354#gh-1370039423:-NhaRvwv40xcnp9pMghr:bsyvgl3) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/15ab0fa47ec399e7a504feef74076b7e92c2d00e/ios/MullvadVPN/View%20controllers/Alert/AlertViewController.swift#L195)):*
<details><summary><i>Previously, buggmagnet wrote…</i></summary><blockquote>

I was wondering what was the difference, it turns out there's an extra space at the end of the `if` statement.

Thankfully, @rablador did run `swiftformat` right, _right_ ? ಠ\_ಠ
</blockquote></details>

Err, sure he did. Never missed it once!

**Review by buggmagnet — APPROVED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5354)* status: all files reviewed, 2 unresolved discussions (waiting on @MrChocolatine)

**Pull request merged**